### ISSUE TITLE
multi-label modelling

### ISSUE THREAD

**Issue by RayKwame** [RayKwame/TheFluShot#22]
Investigate and  understand how it works

[Neural Network Multilabel classifier](https://machinelearningmastery.com/multi-label-classification-with-deep-learning/)

[sk learn MultiOutputClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.multioutput.MultiOutputClassifier.html)
